libr/parse/filter.c

```diff
@@ -148,6 +148,9 @@ static bool filter(RParse *p, ut64 addr, RFlag *f, RAnalHint *hint, char *data,
 	RAnalFunction *fcn;
 	RFlagItem *flag;
 	ut64 off;
+	int bits = p->analb.anal->bits;
+	int seggrn = p->analb.anal->seggrn;
+	int segoff = 0;//r_config_get_i(p->analb.anal->coreb.core, "asm.segoff"); //TODO: respect segoff
```

Comment:
use `#if` for commenting out code

```suggestion
#if 1
int segoff = 0;
#else
// TODO: respect segoff
r_config_get_i (p->analb.anal->coreb.core, "asm.segoff");
#endif
```